docs/v3/core/index.rst

```diff
@@ -440,47 +460,63 @@ Array metadata
 --------------
 
 Each Zarr array in a hierarchy must have an array metadata document, named
-``zarr.json``. This document must contain a single object with the following
+``zarr.json``.
+
+Mandatory
+^^^^^^^^^
+
+This document must contain a single object with the following
 mandatory names:
 
+.. _array-metadata-zarr-format:
+
 ``zarr_format``
-^^^^^^^^^^^^^^^
+""""""""""""""""
 
     An integer defining the version of the storage specification to which the
     array store adheres, must be ``3`` here.
 
+.. _array-metadata-node-type:
+
 ``node_type``
-^^^^^^^^^^^^^^^
+"""""""""""""""
 
     A string defining the type of hierarchy node element, must be ``array``
     here.
```

Comment:
```suggestion
    A string denoting the type of hierarchy node element. It must be the value ``"array"``.
```

Reply:
I noted this in https://github.com/zarr-developers/zarr-specs/issues/329. Thanks.